src/Tribe/Settings.php

```diff
@@ -295,45 +295,51 @@ public function addNetworkPage() {
 		 * @return void
 		 */
 		public function initTabs() {
-			if ( isset( $_GET['page'] ) && $_GET['page'] == $this->adminSlug ) {
-				// Load settings tab-specific helpers and enhancements
-				Tribe__Admin__Live_Date_Preview::instance();
-
-				do_action( 'tribe_settings_do_tabs' ); // this is the hook to use to add new tabs
-				$this->tabs       = (array) apply_filters( 'tribe_settings_tabs', array() );
-				$this->allTabs    = (array) apply_filters( 'tribe_settings_all_tabs', array() );
-				$this->noSaveTabs = (array) apply_filters( 'tribe_settings_no_save_tabs', array() );
-				if ( is_network_admin() ) {
-					$this->defaultTab = apply_filters( 'tribe_settings_default_tab_network', 'network' );
-					$this->currentTab = apply_filters( 'tribe_settings_current_tab', ( isset( $_GET['tab'] ) && $_GET['tab'] ) ? esc_attr( $_GET['tab'] ) : $this->defaultTab );
-					$this->url        = apply_filters(
-						'tribe_settings_url', add_query_arg(
-							array(
-								'page' => $this->adminSlug,
-								'tab'  => $this->currentTab,
-							), network_admin_url( 'settings.php' )
-						)
-					);
-				}
-				if ( ! is_network_admin() ) {
-					$tabs_keys        = array_keys( $this->tabs );
-					$this->defaultTab = in_array( apply_filters( 'tribe_settings_default_tab', 'general' ), $tabs_keys ) ? apply_filters( 'tribe_settings_default_tab', 'general' ) : $tabs_keys[0];
-					$this->currentTab = apply_filters( 'tribe_settings_current_tab', ( isset( $_GET['tab'] ) && $_GET['tab'] ) ? esc_attr( $_GET['tab'] ) : $this->defaultTab );
-					$this->url        = apply_filters(
-						'tribe_settings_url', add_query_arg(
-							array(
-								'page' => $this->adminSlug,
-								'tab'  => $this->currentTab,
-							),
-							admin_url( self::$parent_page )
-						)
-					);
-				}
-				$this->fields_for_save = (array) apply_filters( 'tribe_settings_fields', array() );
-				do_action( 'tribe_settings_after_do_tabs' );
-				$this->fields = (array) apply_filters( 'tribe_settings_fields', array() );
-				$this->validate();
+			if (
+				! isset( $_GET['page'] )
```

Comment:
Personal preference here but I will change `! isset` with a `empty` as removes the need to use a `!` operator

Reply:
I like empty(), too. Updated.